en/user_guide/user.rst

```diff
@@ -0,0 +1,42 @@
+User, Group and Role
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+Page users in administration allow to search, create, edit and delete site users.
```

Comment:
You could also add that users are in the same, front or back and they appear in the same list